pkg/kubernetes/views/registry-dashboard-page.html

```diff
@@ -111,10 +111,45 @@ <h2 class="card-pf-title">
                 <span translatable="yes">Your login credentials do not give you access to use the docker registry from the command line.</span>
             </div>
 <code ng-if="settings.registry.password">$ sudo docker login -p {{settings.registry.password}} -e unused -u unused <span ng-class="{ placeholder: !settings.registry.host_explicit }">{{settings.registry.host}}</span></code>
-            <p translatable="yes">Push an image:</p>
+            <p translatable="yes">Log into OpenShift:
+                <a ng-if="settings.registry.password"
+                    tabindex="0" role="button" popover-trigger="focus"
+                    popover="The registry backend is managed
+                    by an OpenShift server. The oc command line
+                    tool provides advanced administrative control
+                    of the registry.">
+                    <span class="fa fa-lg fa-info-circle"></span>
+                </a>
+            </p>
+<code ng-if="settings.registry.password">$ oc login --token {{settings.registry.password}} <span ng-class="{ placeholder: !settings.registry.openshifthost_explicit }">{{settings.registry.openshifthost}}</span></code>
```

Comment:
Should this default to the current connection url and only use the value of OPENSHIFT_OAUTH_PROVIDER_URL when present?